CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.1.36
```

Comment:
Can be a +1 release – no new features or anything

Reply:
This is just a placeholder for the next real release.  I just haven't updated the Changelog with details. 👍